<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiloteRobotMi10</class>
 <widget class="QMainWindow" name="PiloteRobotMi10">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>701</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>false</underline>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>PiloteRobotMi10</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_quitter">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>570</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Mot DO 10000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Mot DO 10001</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot10000hex">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot10001hex">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot10000bin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot10001bin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_initUOP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IMST + HOLD + SFSPD + ENBL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_faultReset">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FAULT RESET</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RSR">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RSR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Mot DI 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Mot DI 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot1hex">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot0bin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot0hex">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot1bin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resetUOP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset UOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>610</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Statut Modbus Lecture:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statusModbusLecture">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>610</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>6</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
      <underline>true</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Lecture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>276</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
      <underline>true</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Ecriture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>276</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
      <underline>true</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Cycle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_DI1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>115</number>
    </property>
    <property name="maximum">
     <number>128</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stopLecture">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP Lecture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>DI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_appliquerDI1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Mettre à 1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_DI2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>115</number>
    </property>
    <property name="maximum">
     <number>128</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_appliquerDI2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Mettre à 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>DI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot10000label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>961</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mot10001label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>961</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statusModbusEcriture">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>640</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>??</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Statut Modbus Ecriture:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_START">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_HOLD">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HOLD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CSTOPI">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CSTOPI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
